tests/foreman/api/test_syncplan.py

```diff
@@ -167,9 +175,52 @@ def test_create_sync_plan_with_invalid_random_name(self, name):
         @Feature: SyncPlan
 
         """
+        with self.assertRaises(HTTPError):
+            entities.SyncPlan(
+                name=name,
+                organization=self.org,
+            ).create()
 
+    @data(
+        '',
+        ' ',
+        None,
+    )
+    def test_create_sync_plan_with_no_enabled_value_negative(self, enabled):
+        """@Test: Create a sync plan with empty 'enabled' field.
 
-@stubbed()
+        @Assert: A sync plan can not be created with empty 'enabled' field.
+
+        @Feature: SyncPlan
+
+        """
+        with self.assertRaises(HTTPError):
+            entities.SyncPlan(
+                enabled=enabled,
```

Comment:
Is the intent in this test to provide an invalid value for 'enabled', or to provide no value at all? It's possible to simply omit the "enabled" field like so:

```python
entities.SyncPlan(
    interval=…,
    name=…,
    organization=…,
    sync_date=…,
).create(False)  # create_missing=False
```